contracts/StablecoinConverter.sol

```diff
@@ -224,23 +226,81 @@ contract StablecoinConverter is EpochTokenLocker {
         tokenConservation.checkTokenConservation();
         documentTrades(batchIndex, owners, orderIds, volumes, tokenIdsForPrice);
     }
+    /**
+     * Public View Methods
+     */
+
+    /** @dev View returning ID of listed tokens
+      * @param addr address of listed token.
+      * @return tokenId as stored, via BiMap, within the contract.
+      */
+    function tokenAddressToIdMap(address addr) public view returns (uint16) {
+        return IdToAddressBiMap.getId(registeredTokens, addr);
+    }
+
+    /** @dev View returning address of listed token by ID
+      * @param id tokenId as stored, via BiMap, within the contract.
+      * @return address of (listed) token
+      */
+    function tokenIdToAddressMap(uint16 id) public view returns (address) {
+        return IdToAddressBiMap.getAddressAt(registeredTokens, id);
+    }
 
+    /** @dev View returning all currently stored, byte-encoded sell orders
+      * @return encoded bytes representing all orders ordered by (user, index)
```

Comment:
Should we elaborate on the encoding? Or is this just writing the code again?

Reply:
Two options here:

1. These return strings should be concise and we wouldn't want to write the code again...
2. If we are to generate documentation based on this, then perhaps we had better see what the auto generated documentation looks like and decide if it is worth rewriting the code again.

I'd say that its best to be concise and keep these doc-strings to at most on line per @param/return.